insta/src/components/PostContainer/Post.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+
+const Post = props => {
+    return (
```

Comment:
great work. Here would be a great place to use object destructoring. const Post = ({posts}) => 
then you could just use post.properity instead of props.post.property.